abm/server.py

```diff
@@ -169,14 +176,22 @@ def do_put(self, context, descriptor, reader, writer):
         with Config(self.config_path) as config:
             asset_conf = config.for_asset(asset_name)
             connector = GenericConnector(asset_conf, logger, self.workdir)
-            command, catalog = connector.create_write_command(schema)
+            # TODO: Params to the Airbyte objects, such as destination_sync_mode, can be configurable using the arrow-flight request
+            streams=[ConfiguredAirbyteStream(destination_sync_mode=DestinationSyncMode.append,sync_mode=SyncMode.full_refresh,
+               stream=AirbyteStream(name=stream_name,supported_sync_modes=[SyncMode.full_refresh],json_schema=json_schema))]
+            command, catalog = connector.create_write_command(ConfiguredAirbyteCatalog(streams=streams).json())
             socket, container = connector.open_socket_to_container(command)
             idx = 0
             record_reader = reader.to_reader()
+            current_time = time.time() * 1000
```

Comment:
Check if 1000 is needed

Reply:
it is used in airbye code - https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/config_observation.py#L73